pkg/util/config.go

```diff
@@ -271,3 +272,7 @@ func DefaultSystemStrategy() *slov1alpha1.SystemStrategy {
 		WatermarkScaleFactor: pointer.Int64Ptr(150), // 1 means 1/10000
 	}
 }
+
+func DefaultExtensions() *slov1alpha1.ExtensionsMap {
+	return &slov1alpha1.ExtensionsMap{}
```

Comment:
the default value of slo.ExtMap should be extensible(e.g. new plugin can define its own default value in independent files)
consider define as var and `DefaultExtensions()` return the var value

Reply:
// config.go
var defautExtensions = &slov1alpha1.ExtensionsMap{}

// ack_dsa_config.go
var dsaCfg = {}
defautExtensions[dsaDefault] = dsaCfg

// contorller
Regist(dsaCfg)